presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -355,11 +356,13 @@ public void testTableNameAmbiguity()
         session.execute("CREATE TABLE keyspace_4.\"TaBlE_4\" (column_4 bigint PRIMARY KEY)");
         session.execute("CREATE TABLE keyspace_4.\"tAbLe_4\" (column_4 bigint PRIMARY KEY)");
 
+        Thread.sleep(5000);
+
         // Although in Presto all the schema and table names are always displayed as lowercase
         assertContainsEventually(() -> execute("SHOW TABLES FROM cassandra.keyspace_4"), resultBuilder(getSession(), createUnboundedVarcharType())
                 .row("table_4")
                 .row("table_4")
-                .build(), new Duration(1, MINUTES));
+                .build(), new Duration(2, MINUTES));
```

Comment:
Why is this being bumped up?

Reply:
okay this is not required , I shall revert back as 500ms would do the refreshing